src/components/atoms/SpecialServiceCard/SpecialServiceCard.module.scss

```diff
@@ -0,0 +1,25 @@
+@use "sass:map";
+@import "../../../foundations/sass/all";
+
+.service-card {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 15px;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+
+  &__Title {
+    color: rgb(177, 6, 6);
```

Comment:
We should consider using colors from our foundations/colors file instead of using them directly. This would help us to easily change colors in the future.

Reply:
sure